ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_searching_sorting.py

```diff
@@ -337,3 +337,42 @@ def test_jax_numpy_sort_complex(
         a=x[0],
         test_values=False,
     )
+
+
+# where
+@handle_frontend_test(
+    fn_tree="jax.numpy.where",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("numeric")),
+    test_with_out=st.just(False),
+    inplace=st.just(False),
+)
+def test_jax_numpy_where(
+    *,
+    dtype_and_x,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    input_dtype, x = dtype_and_x
+    x = helpers.get_frontend_ret(frontend, f"{frontend}.array", x[0], dtype=input_dtype)
```

Comment:
instead of this you can just do `x= x[0]`
since `x[0]` is a numpy array and thats what our `test_frontend_function` accepts

Reply:
Thanks, I have updated it